pipeline/src/builder.rs

```diff
@@ -189,6 +192,11 @@ impl CommandBuilder {
         self
     }
 
+    pub fn rust_cfgs(mut self, flags: &[&str]) -> Self {
+        self.rust_cfgs = Some(to_strings(flags));
```

Comment:
needs append?

Reply:
append is in inner_command(), which combines the rust_cfgs & rust_flags to CARGO_ENCODED_RUSTFLAGS